repos/system_upgrade/el7toel8/actors/checkpcidevices/actor.py

```diff
@@ -0,0 +1,39 @@
+from leapp.actors import Actor
+from leapp.models import CheckResult, PCIDevices
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+
+
+class CheckPCIDevices(Actor):
+    name = 'check_pci_devices'
+    description = """
+        Some PCI devices does not have necessary kernel module available in Red Hat Enterprise Linux 8 what would
+        prevent such device from working properly. At the time of writing this actor, LSI Logic SCSI Controllers
+        suffer of such problem and if it's present on target system, upgrade process should not be executed.
+        Test case table:
```

Comment:
I guess that this doesn't really belong here, we should figure out where to put this kind of data and how to keep them in sync with the actual code.

Reply:
@shaded-enmity / @bocekm Until we find out the best place for this sort of data, should we keep it here?

Reply:
Yeah, until we agree on where to put it, let's keep it here. We'll know that by the end of next sprint (RHELLEAPP-974) ;)